modules/data.atmosphere/DESCRIPTION

```diff
@@ -57,12 +57,11 @@ Imports:
 Suggests:
     doParallel,
     foreach,
+    geonames,
+    nneo,
     parallel,
     progress,
     rlang (>= 0.2.0)
-Remotes:
-    github::ropensci/geonames,
```

Comment:
We still need the GitHub version geonames until they do another CRAN release -- see #1941 for details.

Reply:
Come to think of it this means we should be specifying the import as `geonames > 0.998`, but they haven't yet bumped the version string in the dev version. I've submitted a PR to geonames and will update here if/when they accept it.

Reply:
geonames version bump PR: https://github.com/ropensci/geonames/pull/18